### ISSUE TITLE
License

### ISSUE THREAD

**Issue by icecreammatt** [ProteusVR/SteamVR_Template#8]
@ProteusVR what is the License for this project MIT / GPL?

**Comment by ProteusVR** [comment]
MIT

**Comment by icecreammatt** [comment]
Thanks, do you mind adding a license file to the repo? I can send you a PR if you want.

**Comment by ProteusVR** [comment]
Yes you have a model for the license? I can merge it no problems.

**Comment by icecreammatt** [comment]
@ProteusVR Here's a PR https://github.com/ProteusVR/SteamVR_Template/pull/9

Also thanks for sharing these great resources! :+1:
